src/bundle/Resources/public/scss/_preview.scss

```diff
@@ -0,0 +1,87 @@
+.ez-preview {
+    .ez-preview__nav {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        background-color: $ez-ground-primary;
+        padding: .2rem 1rem;
+
+        .ez-preview__item {
+            flex-basis: 45%;
+
+            &--back {
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+                line-height: 0;
+            }
+
+            &--actions {
+                flex-basis: auto;
+                display: flex;
+                justify-content: center;
+                border: 1px solid $ez-color-base-medium;
+                border-radius: 5px;
+                padding: .1rem;
```

Comment:
`.1rem` is `0.0625px` if font size is set to `16px`. Maybe it should be `0.125rem` instead, which translates to `1px`? Some browsers can do subpixel rendering.